backend/app/crawler.py

```
@@ -1,4 +1,4 @@
-from typing import List, Optional, Set
+from typing import List, Optional, Set, Dict
```

Ruff (0.8.2)

`typing.List` is deprecated, use `list` instead

(UP035)